source/Table/Table.js

```diff
@@ -453,13 +453,15 @@ export default class Table extends PureComponent {
       a11yProps['aria-describedby'] = id;
     }
 
+    a11yProps.key = "Row" + rowIndex + "-" + "Col" + columnIndex;
+    a11yProps.className = cn('ReactVirtualized__Table__rowColumn', className);
+    a11yProps.style = style;
+    a11yProps.title = title;
+    
     return (
       <div
         {...a11yProps}
-        key={`Row${rowIndex}-Col${columnIndex}`}
-        className={cn('ReactVirtualized__Table__rowColumn', className)}
-        style={style}
-        title={title}>
+       >
```

Comment:
Looks like the crux of this fix is to avoid generating sub-optimal `React.createElement` boilerplate with an added function call and `for`-loop. But that's not obvious at a glance, and might regress later.

I'd suggest adding a comment about why it's important not to mix inline props and spread props.